src/traces/ohlc/attributes.js

```diff
@@ -102,7 +102,7 @@ module.exports = {
         valType: 'number',
         min: 0,
         max: 0.5,
-        dflt: 0.05,
+        dflt: 0.1,
```

Comment:
Sure, either 0.2 or 0.3 is OK. I still sort of prefer 0.3... but having never used these charts myself it's not a very informed opinion.